Topic1_Arrays/Day2207/Tung/1.txt

```diff
@@ -0,0 +1,25 @@
+// Description: Use two Loop
+// Time complexity: O(n ^ 2)
+// Memory complexity: O(1)
+
+class Solution {
+public:
+    vector<int> twoSum(vector<int>& nums, int target) {
+        vector<int> kq ;
```

Comment:
Use English variable name. Sample "result"
Try to find better solution instead of brute force solution